lib/slide_hub/cloud/engine/azure.rb

```diff
@@ -1,6 +1,8 @@
 require 'azure'
-require 'azure-contrib'
+# require 'azure-contrib'
 require 'json'
+require "azure/storage"
```

Comment:
__[RuboCop]__

Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.


<!-- Comment by [SideCI](https://sideci.com) -->